client/my-sites/site-settings/seo-settings/main.jsx

```diff
@@ -12,34 +11,10 @@ import notices from 'notices';
 import QuerySitePurchases from 'components/data/query-site-purchases';
 import QuerySiteSettings from 'components/data/query-site-settings';
 import { getSitePurchases, hasLoadedSitePurchasesFromServer, getPurchasesError } from 'state/purchases/selectors';
-import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSelectedSiteId, getSelectedSite } from 'state/ui/selectors';
 import SeoForm from './form';
```

Comment:
yes this is a particularly difficult thing to port and we have existing UI quirks because of how the extended settings get or don't get updated.